Discripsions of Scripts:
The Linux script "1.Filter&mapping&count.sh" was used to process the RNA-seq raw data and obtain expression information.
The R scripts "2.Remove_BatchEffect.r", "3.DEanalysis.r", "4.Exp_evolution_test.r" and "5.GO_enrichment.r" were used to remove batch effect, do the differential expression analysis, do the HEB, cis-trans as well as ELD analysis, and do the GO enrichment, respectively.
The homoeologous relationships are from Hu et al. 2024.

Experimental Design:
RNA-seq libraries were generated for two allopolyploid species (AD genome, 2n=4x=52), G. hirsutum cv. Jin668 (AD1) and G. barbadense acc. VSg-1 (AD2), their derived haploid lines (hAD genome, n=2x=26), models of their A- and D-genome diploid progenitors, G. arboreum (A2) and G. raimondii (D5), as well as their synthetic F1 hybrid. Three to six biological replicates.
